src/theme/variables.css

```diff
@@ -264,4 +264,15 @@ http://ionicframework.com/docs/theming/ */
   .mobile-only {
     display: none;
   }
+}
+.empty-state {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  align-items: center;
+}
+.empty-state ion-spinner{
+  margin-right: 10px;
```

Comment:
Find some other way to fix this, we don't need to update the styling of empty-state class.